cdc/processor/pipeline/sink.go

```diff
@@ -151,9 +154,12 @@ func (n *sinkNode) flushSink(ctx context.Context, resolvedTs model.Ts) (err erro
 		return errors.Trace(err)
 	}
 	checkpointTs, err := n.sink.FlushRowChangedEvents(ctx, n.tableID, resolvedTs)
-	if err != nil {
+	// we can ignore ErrFlushTsBlocking error because resolvedTs will continue
+	// to come
```

Comment:
Do you mean `flushSink` is called periodically when resolvedTs is updated?

Reply:
Yes, it is what I mean, I will polish this comment.